central/Dockerfile

```diff
@@ -17,7 +17,8 @@ COPY --from=builder --chown=glowroot:root /tmp/glowroot-central /usr/share/glowr
 
 WORKDIR /usr/share/glowroot-central
 
-RUN chmod a+x /usr/local/bin/docker-entrypoint.sh \
+RUN chmod g+w /usr/share/glowroot-central \
```

Comment:
something like below, just worried this will be easy to regress/forget why
```suggestion
# need to give group write permission because OpenShift will ignore USER directives
# and run the POD as a numbered userid which cannot possibly exist on the underlying physical host
# but which is guaranteed to be a member of the "root" group
RUN chmod g+w /usr/share/glowroot-central \
```